native/src/utils/getNavigationDeepLinks.ts

```diff
@@ -0,0 +1,11 @@
+import { Platform } from 'react-native'
+
+/**
+ * Depending on the platform this function returns the correct deeplink for navigation app
+ * @param address: location address
+ */
+export const getNavigationDeepLinks = (address: string): string | undefined =>
+  Platform.select({
+    ios: `maps:0,0?q=${address}`,
+    android: `geo:0,0?q=${address}`
```

Comment:
Maybe we should specify long/lat as well: https://developers.google.com/maps/documentation/urls/android-intents#categorical-search

Reply:
added coordinates --> worked on Maps.me